WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java

```diff
@@ -680,27 +688,31 @@ private void enterCharges()
         enterChargesInGrid(1, mapWithAnimalId);
 
         log("Submit the form");
+        sleep(5000);
```

Comment:
We try hard to avoid sleep(). Has anyone investigated how we can make this more targeted?

@labkey-sweta @labkey-bpatel

Reply:
Looks like sleep() is still used when testing EHR data entry.

Reply:
OK to proceed with the merge, but I'm definitely interested in how we can remove them as we touch tests going forward.